main.go

```diff
@@ -47,16 +86,19 @@ func main() {
 			http.Error(w, "Internal server error", http.StatusInternalServerError)
 			slog.Error("Failed parsing darkmode status", "error", err)
 		}
-		systemsToShow := systems
+
+		whichLinks := strings.Split(r.Host, ".")[0]
```

Comment:
I agree that it should be possible to control what version is shown besides looking at the subdomain - how was I supposed to even review this PR otherwise? I had to override the Host header to test it.

Even if you want to look at the subdomain, have an escape hatch like a `FORCE_VERSION=sm` envvar or a `?version=sm` GET param that takes higher priority than the subdomain. (What's the subdomain of localhost?)

Reply:
On most computers *.localhost points to 127.0.0.1, so e.g. http://sm.localhost:3000/ should work

Reply:
But this should definitely be documented